extensions/EXT/SPV_EXT_optnone.asciidoc

```diff
@@ -0,0 +1,124 @@
+SPV_EXT_optnone
+===============
+
+Name Strings
+------------
+
+SPV_EXT_optnone
+
+Contact
+-------
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Headers
+
+Contributors
+------------
+
+- Mariya Podchishchaeva, Intel
+- Ben Ashbaugh, Intel
+- Alexey Sotkin, Intel
+- Joe Garvey, Intel
+- Dmitry Sidorov, Intel
+
+Notice
+------
+
+Copyright (c) 2024 The Khronos Group Inc. Copyright terms at http://www.khronos.org/registry/speccopyright.html
+
+Status
+------
+* Complete
+
+Version
+-------
+
+[width="40%",cols="25,25"]
+|========================================
+| Last Modified Date | 2024-06-21
+| Revision           | 2
+|========================================
+
+Dependencies
+------------
+
+This extension is written against the SPIR-V Specification,
+Version 1.6 Revision 3, Unified
+
+This extension requires SPIR-V 1.0.
+
+Overview
+--------
+
+This extension adds new possible value for Function Control mask - *OptNoneEXT*,
+which represents a strong request to not optimize the function. It is useful in cases
+where the user wants to debug a particular function without sacrificing performance of
+the entire application, and this is accomplished by disabling optimizations solely
+for that particular function instead of the entire module.
+
+Extension Name
+--------------
+
+To use this extension within a SPIR-V module, the following
+*OpExtension* must be present in the module:
+
+----
+OpExtension "SPV_EXT_optnone*
+----
+
+Modifications to the SPIR-V Specification, Version 1.6, Revision 3, Unified
+---------------------------------------------------------------------------
+Capabilities
+~~~~~~~~~~~~
+
+Modify Section 3.31, "Capability", adding these rows to the Capability table:
+
+--
+[options="header"]
+|====
+2+^| Capability ^| Implicitly Declares
+| 6094 | *OptNoneEXT* +
+Allow to use *OptNoneEXT* Function Control mask value |
```

Comment:
Confirming: there are no concerns using **OptNoneEXT** as both the name of the Capability and the name of the Function Control bit?